src/propertymanipulation.jl

```diff
@@ -0,0 +1,7 @@
+### Property manipulation ###
```

Comment:
Can't these live in the highlevel file? I don't think it's helpful to have these in a separate file.

Reply:
Sure, why not?